### PR TITLE
Adding option to smooth/unsmooth spectrum in Igmguesses

### PR DESCRIPTION
**Pull request by marijana777**
* - smooth spectrum, / - unsmooth spectrum

### CODE REVIEW

File: pyigm/guis/igmguesses.py [hunk 1]
```diff
@@ -616,6 +617,7 @@ def __init__(self, ispec, z, parent=None, llist=None, norm=True, screen_scale=1.
         self.parent = parent
         self.spec = ispec
         self.spec_fil = self.spec.filename
+        self.orig_spec = ispec  # For smoothing
```
Comment (profxj) [inline pyigm/guis/igmguesses.py]: You may need to use `.copy()` here

**Review by profxj — COMMENTED** [review]
Consider the comments

File: pyigm/guis/igmguesses.py (hunk 1 above)
Comment (profxj) [reply pyigm/guis/igmguesses.py]: And `.copy()` when you re-assign `self.spec` for un-smoothing.

**Comment by ntejos** [comment]
Thanks @marijana777 for this. How about using the same keystrokes for smoothing/unsmoothing as in the linetools' GUIs? e.g lt_xspec?

File: pyigm/guis/igmguesses.py (hunk 1 above)
Comment (marijana777) [reply pyigm/guis/igmguesses.py]: Thanks @profxj . Now using .copy(), in both cases. (Also, continuum-normalized orig_spec, and defined bad_pixels.)

**Comment by marijana777** [comment]
Thanks @ntejos . I think that this would be a good idea. 
However, U and S are already used in different way in igmguesses (U - restrict to subset of available lines from the parent LineList at current redshift, S - select an absorption component from cursor position)

**Comment by profxj** [comment]
merging.

**Pull request merged**